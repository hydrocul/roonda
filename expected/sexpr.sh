echo Hello
echo Hello World
echo 'Hello   World!'
echo roonda_0ac56896a1a3f1427f602202defd62faaa1cc4c8.txt

#################################################
# roonda_0ac56896a1a3f1427f602202defd62faaa1cc4c8.txt:
#################################################
# 
# Hello Hello
# 
