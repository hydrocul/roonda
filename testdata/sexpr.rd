(sh)
(echo Hello)
(echo Hello   World)
(echo "Hello   World!")

(echo attachedfile.txt)

<< attachedfile.txt

Hello Hello

attachedfile.txt >>

