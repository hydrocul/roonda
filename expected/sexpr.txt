Hello
Hello World
Hello   World!
roonda_0ac56896a1a3f1427f602202defd62faaa1cc4c8.txt
